internal/cmd/types/file.go

```diff
@@ -22,10 +23,24 @@ func (s *StringOrFile) String() string {
 
 func (s *StringOrFile) Set(raw string) error {
 	fh, err := os.Open(raw)
-	if err != nil {
-		logging.L.Debug().Err(err).Msg("failed to open file")
+	switch {
+	case errors.Is(err, os.ErrNotExist):
+		limit := 10
+		switch {
+		case len(raw) <= 5:
+			limit = 0
+		case len(raw) <= 10:
+			limit = 5
```

Comment:
maybe always show 50% of the characters?

Reply:
Ya, that seems easier, and should be fine. I'll do that